### PR TITLE
fake data for seeding db

### PR DESCRIPTION
**Pull request by ikidnapmyself**

### CODE REVIEW

File: src/DataFixtures/UserFixtures.php [hunk 1]
```diff
@@ -12,9 +12,11 @@ class UserFixtures extends Fixture implements OrderedFixtureInterface
     const TEST_USER = 'test-user';
     public function load(ObjectManager $manager)
     {
+        $faker = \Faker\Factory::create();
+
         $user = new User();
-        $user->setUsername('Aurora');
-        $user->setEmail('user@test.com');
+        $user->setUsername($faker->userName);
+        $user->setEmail($faker->email);
```
Comment (bencagri) [inline src/DataFixtures/UserFixtures.php]: this should be static email.

**Review by bencagri — CHANGES_REQUESTED** [review]
Could you check the changes?

File: src/DataFixtures/UserFixtures.php (hunk 1 above)
Comment (ikidnapmyself) [reply src/DataFixtures/UserFixtures.php]: I see your point.

**Review by bencagri — APPROVED** [review]
Approved

**Pull request merged**